### PR TITLE
Update make-configmgr-collection-available-edr.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated step 3 to add AAD group

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by Brenduns** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@Brenduns When you sign off then please either merge the PR or leave it as it is so that @tiburd can procced to merge. Please close the PR only if you disagree and do not want the changes to be reflected in the document Tina your comment please.

File: memdocs/intune/protect/includes/make-configmgr-collection-available-edr.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ ms.date: 12/18/2020
   
    ![Configure cloud sync](../media/tenant-attach-intune/cloud-sync.png)
 
-3. Select **OK** to save the configuration.
+3. Click on **Add** and **Select** the Azure Active Diretiry group that you would like to syncrhonize Collect memberhsip results with.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/includes/make-configmgr-collection-available-edr.md:19]:
```suggestion
3. Select **Add** and then select the Azure Active Directory group that you would like to synchronize with _Collect membership results_.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger7** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tiburd** [comment]
> @Brenduns When you sign off then please either merge the PR or leave it as it is so that @tiburd can procced to merge. Please close the PR only if you disagree and do not want the changes to be reflected in the document Tina your comment please.

Yes, that is how the process usually works. 
Brent, thank you for reviewing and approving the proposed changes. You're welcome to contact me if you have any questions about the process. 
Thanks!

**Pull request merged**